Helper/Webhook.php

```diff
@@ -1208,11 +1208,13 @@ private function setPrePaymentAuthorized()
             'adyen_abstract',
             $this->order->getStoreId()
         );
+        // Possible states entered by the configured status
+        $possibleStates = ['new', 'processing'];
```

Comment:
I would propose to move these states and the finalizeOrder ones to a constant multidimensional array in this class